src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    const animal = Animal.alive;
+    const index = animal.indexOf(this);
+
+    if (animal !== 0) {
+      animal.splice(index, 1);
+    }
+  }
 }
```

Comment:
I like the solution, but
1. `Animal.alive` is an array of animals and you save it to variable `animal`
2. `if (animal !== 0)` this is always true because as I said this is an array.